library/std/src/io/tests.rs

```diff
@@ -823,3 +823,20 @@ fn try_oom_error() {
     let io_err = io::Error::from(reserve_err);
     assert_eq!(io::ErrorKind::OutOfMemory, io_err.kind());
 }
+
+#[test]
+#[cfg(all(windows, unix, not(miri)))]
```

Comment:
This cfg is never true because it expects a platform that is both `windows` and `unix`

Reply:
Ahhh damn thanks for spotting this

Reply:
Raised in #135635